### PR TITLE
feat: date/time picker improvements

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* Updates the DS to 1.8.0 to implement the Date/Time Picker updates as Comboboxes

### Why is it needed?

* Because typing dates & times just makes sense

### How to test it?

* automated testing 🪄 

### Related Issues

* resolves CONTENT-195
* https://github.com/strapi/design-system/pull/985

### CODE REVIEW

**Comment by joshuaellis** [comment]
NOTE: I cannot, for the life of me figure out why those two tests fail considering they pass in the DS and are the same & the work in the actual app. Something to investigate before merging.

File: examples/getstarted/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "strapi": "strapi"
   },
   "dependencies": {
-    "@strapi/icons": "1.7.9",
+    "@strapi/icons": "0.0.0-3cf3ff39",
```
Comment (jhoward1994) [inline examples/getstarted/package.json:15]: Is this version correct? Should it not be 1.8.0 ?

Comment (gu-stav) [reply examples/getstarted/package.json:15]: No the DS is currently released as an experimental release and the PR is still open.

Comment (gu-stav) [reply examples/getstarted/package.json:15]: For reference https://github.com/strapi/design-system/pull/985

**Review by jhoward1994 — APPROVED** [review]

**Pull request merged**